thrift/tchannel.thrift

```diff
@@ -0,0 +1,8 @@
+struct ClientLibrary {
```

Comment:
Endoints are cheap. Might as well put everything in meta and expose meta on all service names including tchannel.